### ISSUE TITLE
[issue] when watching second episode, strange UI appears

### ISSUE THREAD

**Issue by TobiPeterG** [TheGoddessInari/netflix-1080p-firefox#12]
Hey guys,

This add-on works great for me, I just have one issue.
When I watched an episode, the outro starts, the "next episode" button appears with a timer for the next episode and when the next episode starts, the following UI appears in the center of the screen:
![IMG_20210106_230345.jpg](https://user-images.githubusercontent.com/19935382/103824467-40e35200-5074-11eb-9b72-e0c552ff29f4.jpg)

The only way I found to make it to away is to press on "Reset", the add-on still works and now it doesn't appear anymore, even when the next episode starts.

I'm on latest Firefox (84), can someone confirm this issue?

**Comment by cheoppy** [comment]
Yes, I've seen it too in the last week. I usually just restart the playback and it disappears.

**Comment by Termuellinator** [comment]
the shortcut for this is alt+ctrl+shift+s , i can't reproduce getting it with the "play next" button though ^^

**Comment by TheGoddessInari** [comment]
Does this occur with "select highest bitrate" checked?

All this extension does as far as that is remove the always-false check. It still requires the webpage to believe that the key combo has been pressed.

**Comment by TobiPeterG** [comment]
> Does this occur with "select highest bitrate" checked?
> 
> All this extension does as far as the bitrate/CDN menu is remove the always-false check. It still requires the webpage to believe that the key combo has been pressed.

Yes, I have that option enabled

**Comment by moneroexamples** [comment]
@TheGoddessInari Yes, the same issue here. If `select highest bitrate` is chosen, the second episode will have this UI present, and you have to manually switch it off.

**Comment by mikeloeven** [comment]
I can confirm this it looks like the issue is triggered by the plugin loading and changing bit rates however something is occurring when the show advances automatically vs manually that prevents the window from closing properly requiring manual intervention
